staging/src/k8s.io/apiserver/pkg/util/flowcontrol/controller_test.go

```diff
@@ -389,22 +389,19 @@ func TestAPFControllerWithGracefulShutdown(t *testing.T) {
 		QueueSetFactory:        cts,
 	})
 
-	stopCh, controllerCompletedCh := make(chan struct{}), make(chan struct{})
-	var controllerErr error
-
-	informerFactory.Start(stopCh)
-
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
+
```

Comment:
In unit tests use `_, ctx := ktesting.NewTestContext(t)` to get per-test output.

Reply:
done